components/admin_console/group_settings/__snapshots__/group_settings.test.jsx.snap

```diff
@@ -27,10 +27,11 @@ For more information on Groups, please see [documentation](!https://about.matter
     </div>
   </div>
   <AdminPanel
+    className=""
```

Comment:
Does it make sense for this property to be required if it's often an empty string?

Reply:
Yes, I don't have an strong opinion here, it's just easier to just have a isRequiered + default than handle it inside. This way, if you don't pass it I don't need to handle the null case.